include/swift/AST/SILOptions.h

```diff
@@ -146,6 +146,9 @@ class SILOptions {
   /// \brief Enable large loadable types IRGen pass.
   bool EnableLargeLoadableTypes = true;
 
+  /// \brief Enable expansion of types regardless of size.
+  bool EnableExpandAll = false;
```

Comment:
Ok. I was just imagining an LLVM option that could be put where-ever you defined expand. But this works as well.